services/integration/src/models/constitution/PreviousInformation.model.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const previousInformationSchema = new Schema({
+  description: {
```

Comment:
Este schema necesita un Ptoject id para poder relacionar esta info con un project especifico

Reply:
really? , yesterday i asked you , and you told me that is not necessary